### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by davidmerfield** [davidmerfield/Typeset#38]
Screen readers stumble on something like ```<span>a</span>t```, reading it as 'A T' instead of 'at'. This makes Blot's hanging punctuation trick difficult to read. This should be possible by tricking the screanreader into reading the word using aria-label or aria-hidden or some combination of the two.

**Comment by davidmerfield** [comment]
Google translate also screws up

**Comment by davidmerfield** [comment]
I've now disabled the optical margin alignment feature by default. It breaks too many things.

**Comment by danielhaim1** [comment]
Can you provide an example where you use Google Translate with Typeset?
Are you testing different language support?

**Comment by danielhaim1** [comment]
I need to think this through and do some research, from both web accessibility and search optimization.

One solution that was discussed is the possibility of adding an `aria-label` to the parent element that's being converted -- but that's too risky (and may require a major version bump that will break some sites).

- If user has set `.post-typeset` to a `<div>` element that already contains `aria-label` it may break things.
- I suppose we can wrap the element with another span that contains the `aria-label`, but I also need to double check that this is the best approach.

I'll be investigating this in the coming days.

**Comment by AndrewKvalheim** [comment]
Would it work to just wrap the whole word instead?

```html
<span class="pull-A">At</span> the beginning
```

**Comment by davidmerfield** [comment]
@AndrewKvalheim Brilliant! I can't think of a good reason that would not work
